### ISSUE TITLE
Error When Sorting Mods

### ISSUE THREAD

**Issue by ghost** [Yacoby/Wrye-Mash#27]
I get the following error when trying to sort my mods from 'Modified' to 'Load Order':

```
Traceback (most recent call last):
  File "D:\games\Morrowind\Mopy\mash\masher.py", line 3079, in Execute
    self.window.PopulateItems(self.sortCol,-1)
  File "D:\games\Morrowind\Mopy\mash\gui\__init__.py", line 112, in PopulateItems
    self.PopulateItem(itemDex,mode,selected)
  File "D:\games\Morrowind\Mopy\mash\masher.py", line 904, in PopulateItem
    if not mosh.mwIniFile.isWellOrdered(fileName):
  File "D:\games\Morrowind\Mopy\mash\mosh.py", line 2354, in isWellOrdered
    return not self.doubleTime[modInfos[loadFile].mtime]
KeyError: 1051401906.0
```

I close the error window, and then I get a messae box telling me that some times will be changed. I click OK, and then get this traceback:

```
Traceback (most recent call last):
  File "D:\games\Morrowind\Mopy\mash\masher.py", line 2435, in RefreshData
    globals.modList.Refresh(popMods) #--Will repop saves too.
  File "D:\games\Morrowind\Mopy\mash\masher.py", line 863, in Refresh
    self.PopulateItems(selected=selected)
  File "D:\games\Morrowind\Mopy\mash\gui\__init__.py", line 112, in PopulateItems
    self.PopulateItem(itemDex,mode,selected)
  File "D:\games\Morrowind\Mopy\mash\masher.py", line 904, in PopulateItem
    if not mosh.mwIniFile.isWellOrdered(fileName):
  File "D:\games\Morrowind\Mopy\mash\mosh.py", line 2354, in isWellOrdered
    return not self.doubleTime[modInfos[loadFile].mtime]
KeyError: 1051401906.0
```

On a side note, why aren't these tracebacks being logged?

Wrye Mash Version: 85-dev-9097a80be8

**Comment by Yacoby** [comment]
Is this easy to reproduce for you? (i.e. does it happen every time)

**Comment by Yacoby** [comment]
Oh, and regarding tracebacks being logged, this should have been fixed by #28

**Comment by Yacoby** [comment]
I can't reproduce this problem, so this is my best guess at a fix for it
